kubernetes/apps/nachet/nachet-backend-deployment.yml

```diff
@@ -38,21 +38,21 @@ spec:
     spec:
       serviceAccountName: secrets-reader
       containers:
-      - name: nachet-backend
-        image: ghcr.io/ai-cfia/nachet-backend:48
-        imagePullPolicy: Always
-        command: ["/bin/sh", "-c"]
-        args:
-        - >
-          cp /vault/secrets/.env . && hypercorn -b :8080 app:app
-        ports:
-        - containerPort: 8080
-        livenessProbe:
-          httpGet:
-            path: /health
-            port: 8080
-          initialDelaySeconds: 60
-          periodSeconds: 10
+        - name: nachet-backend
+          image: ghcr.io/ai-cfia/nachet-backend:48
+          imagePullPolicy: Always
+          command: ["/bin/sh", "-c"]
+          args:
+            - >
+              cp /vault/secrets/.env . && hypercorn -b :8080 app:app
```

Comment:
Can you clarify why the environment variables cannot be passed to the container and why we have to break the container abstraction (directly configuring the Asynchronous Server Gateway Interface webserver to setup the environment? I've asked why this was necessary previously but haven't heard a good explanation to why this is preferable.

Reply:
This PR is only solving yaml linting violatiions and applying the new line length ruling to 120. Can we discuss this in an issue ? From what I understood we want to address this issue [here](https://github.com/ai-cfia/howard/issues/27)